module/migration.mjs

```diff
@@ -29,7 +29,7 @@ export async function migrateWorld({ bypassVersionCheck=false }={}) {
           updateData = foundry.utils.mergeObject(source, updateData, {inplace: false});
         }
         await actor.update(updateData, {
-          enforceTypes: false, diff: valid && !flags.persistSourceMigration, render: false
+          enforceTypes: false, diff: valid && !flags.persistSourceMigration, recursive: false, render: false
```

Comment:
If `updateData` is not the full source, then updating with `recursive: false` might do things like truncate the entire items collection.

```suggestion
          enforceTypes: false, diff: valid && !flags.persistSourceMigration, recursive: !flags.persistSourceMigration, render: false
```